bw2io/extractors/ecospold1.py

```diff
@@ -21,6 +21,24 @@ def getattr2(obj, attr):
 class Ecospold1DataExtractor(object):
     @classmethod
     def extract(cls, path, db_name, use_mp=True):
+        """
+        Extract data from ecospold1 files.
+
+        Parameters
+        ----------
+        path : str
+            Path to the directory containing the ecospold1 files or path to a single file.
+        db_name : str
+            Name of the database.
+        use_mp : bool, optional
+            If True, uses multiprocessing to parallelize extraction of data from multiple files, by default True.
```

Comment:
> by default True

Is this following the [NumPy style guide](https://numpydoc.readthedocs.io/en/latest/format.html)?

Reply:
Revised.